src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -510,7 +584,19 @@ def ZCLReadAttribute(self, cluster, attribute, nodeid, endpoint, groupid, blocki
         result = asyncio.run(self.ReadAttribute(nodeid, [(endpoint, req)]))
         path = ClusterAttribute.AttributePath(
             EndpointId=endpoint, Attribute=req)
-        return im.AttributeReadResult(path=im.AttributePath(nodeId=nodeid, endpointId=path.EndpointId, clusterId=path.ClusterId, attributeId=path.AttributeId), status=0, value=result[path].Data.value)
+        return im.AttributeReadResult(path=im.AttributePath(nodeId=nodeid, endpointId=path.EndpointId, clusterId=path.ClusterId, attributeId=path.AttributeId), status=0, value=result['Attributes'][path].Data.value)
+
+    def ZCLReadEvent(self, cluster, event, nodeid, endpoint, groupid, blocking=True):
```

Comment:
Why do we need this? We should only be supporting the REPL approach going forward.

Reply:
sure, will remove it.

Reply:
removed, thanks